quick-start/e2e/page-objects/flows/flows.ts

```diff
@@ -202,7 +263,7 @@ export class FlowPage extends AppPage {
     // no progress bar before we start
     // expect(element(by.css('.job-progress')).isPresent()).toBe(false);
 
-    browser.wait(EC.elementToBeClickable(this.mlcpRunButton));
+    //browser.wait(EC.elementToBeClickable(this.mlcpRunButton));
```

Comment:
looks like this line can be deleted

Reply:
yes, mlcpRunButton (Run Input Flow button) is always presence